common/models/acl.js

```diff
@@ -462,6 +462,16 @@ module.exports = function(ACL) {
       methodNames,
       registry: this.registry});
 
+    if (!context.isScopeAllowed()) {
+      req.permission = ACL.DENY;
+      debug('--Denied by scope config--');
+      debug('Scopes allowed:', context.accessToken.scopes || 'DEFAULT');
+      debug('Scope required:', context.getScopes() || 'DEFAULT');
+      context.debug();
```

Comment:
Good catch, this should have been using `DEFAULT_SCOPES` for the token, and no `||` clause for context scopes (`getScopes()` should be already handling the default case). I'll fix this ASAP.